src/main/java/teammates/ui/webapi/request/StudentProfileUpdateRequest.java

```diff
@@ -0,0 +1,76 @@
+package teammates.ui.webapi.request;
+
+/**
+ * The request of updating a student profile.
+ */
+public class StudentProfileUpdateRequest extends BasicRequest {
+
+    private String shortName;
+    private String email;
+    private String institute;
+    private String nationality;
+    private String gender;
+    private String moreInfo;
+    private String existingNationality;
```

Comment:
What is the difference between `existingNationality` and `nationality `

Reply:
```
editedProfile.nationality = getRequestParamValue(Const.ParamsNames.STUDENT_NATIONALITY);
        if ("".equals(editedProfile.nationality)) {
            editedProfile.nationality = getRequestParamValue("existingNationality");
        }
```

This is logic in master, I didn't change the logic and keep it the way as it should be.